Dockerfile

```diff
@@ -4,6 +4,8 @@ FROM docker.io/golang:1.20.5-alpine3.17 as builder
 RUN apk update && apk add --no-cache \
     gcc \
     git \
+    # linux-headers are needed for Ledger support
+    linux-headers \
```

Comment:
[note to reviewers] this fixes the Docker workflow. Tested in https://github.com/rootulp/celestia-app/pull/9